### PR TITLE
#119 Change party id type

### PR DESCRIPTION
**Pull request by kmaslowiec**

### CODE REVIEW

File: app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt [hunk 1]
```diff
@@ -128,7 +128,13 @@ private fun PartyTile(
...
-	val parties = (1..5).map {
+	val parties = listOf(
+		"one",
+		"two",
+		"three",
+		"four",
+		"five",
+	).map {
 		Party(
 			id = it,
```
Comment (MarekMacko) [inline app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt]: Just use `it.toString()`

**Review by MarekMacko — CHANGES_REQUESTED** [review]

File: app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt [hunk 2]
```diff
@@ -128,19 +128,14 @@ private fun PartyTile(
...
-		"four",
-		"five",
-	).map {
-		Party(
-			id = it,
-			title = "title$it",
-			description = "description$it",
-		)
-	}
+	val parties = (1..5).map { it.toString() }
```
Comment (MarekMacko) [inline app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt]: There is no need for two consecutive `map` operators, keep it simple and merge them into one

Comment (kmaslowiec) [reply app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt]: Done

Comment (MarekMacko) [reply app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt]: Fix CI errors and we're ready to go!

Comment (kmaslowiec) [reply app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt]: Done

**Review by MarekMacko — APPROVED** [review]

**Pull request merged**